WordPress/Jetpack/AppConfiguration.swift

```diff
@@ -1,6 +1,11 @@
 import Foundation
 
+/**
+ * Jetpack Configuration
+ */
 @objc class AppConfiguration: NSObject {
     @objc static let isJetpack: Bool = true
     @objc static let allowSiteCreation: Bool = false
+    @objc static let jetpackLogin: Bool = true
```

Comment:
I'm wondering if there's a better name for this, but I can't come up with a better solution... 🤔 
I understand it means "login flow for jetpack", but I think it can be kinda confusing if you just see `AppConfiguration.jetpackLogin`

Reply:
Same here! I originally had "jetpackLoginFlow" but that was like meh then I just used "isJetpack" and that didn't feel right either, so I just used "jetpackLogin". It's not great, but I can't think of a better name. 

@leandroalonso any thoughts? 🤔